sag_htt.par

```diff
@@ -7,9 +7,9 @@ fifth_size	703.5787
 oct_size	2/1
 !
 !Lower bound	Symbol	Value		ASCII
--70.385	(!)	704/729	o
+-70.385	(!)	704/729	w
 -62.5	\!/	32/33		v
--55.0	(!)	704/729	o
+-55.0	(!)	704/729	w

```

Comment:
It seems weird to me that the high tolerance temperament has some repeated notes, but I can't find a reference for what should be correct. I plugged the cents values [here](https://github.com/cmloegcmluin/YahooTuningGroupsUltimateBackup/blob/068a580a229d30e6005579ae8cce496e7f12ac36/tuning-math/files/secor/scl/secor17htt1.scl) into the calculator, but didn't discover any correspondence with this .par file though.

Reply:
I don't see any repeated notes. I assume you mean repeated symbols. i.e. (!) and (|). I don't know anything about that, but I assume Scala handles it correctly and gives a sensible result when you notate the HTT scale with the HTT notation. BTW, George agreed when I pointed out recently, that "high tolerance" is a misnomer, but too late to change the name. It is really a "low tolerance temperament" or equivalently a "high precision temperament".

Reply:
I had seen that somewhere re: the name, yes. But I still don't understand what the _HTT_ is, certainly not well enough to ascertain whether this configuration results in Scala notating it correctly. 

I did mean repeated _symbols_, yes. They seem rather strange to me. If you trust it was working before, though, then I suppose we shouldn't fiddle with it.

Reply:
Yah. It was George's baby entirely. I don't know anything about it. But if you want to test it, search for files of the form secor*htt*.scl in the scala archive and load them into scala and SET NOTA SAHTT (or something like that) then see if the notation makes sense.
http://huygens-fokker.org/docs/scalesdir.txt

Reply:
I had looked in there and found the HTT which wasn't named as a subset:

```
! secor29htt.scl
!
George Secor's 29-tone 13-limit high-tolerance temperament (5/4 & 7/4 exact)
 29
!
58.08980
97.04984
140.19633
179.15637
207.15739
265.24719
296.73557
347.35372
5/4
414.31478
472.40458
496.42131
554.51111
593.47114
633.37025
679.56197
703.57869
761.66849
800.62853
843.77502
882.73506
910.73608
7/4
992.84261
1050.93241
1089.89245
1117.89347
1175.98327
2/1
```

Unfortunately it's all in cents. I could try reverse engineering it. I don't happen to see any steps the size of a secor. I can see the third is 703.57869 cents so that should be enough for me to determine whether the symbols Scala comes up with from this config are correct. But it wouldn't be quick, so I'll resolve this for now.

Reply:
Fair enough. I assume you meant "the _fifth_ is 703.57869 cents".

Reply:
I did mean that, yes. 

I'm surprised I didn't notice that there were two fifth sizes, given that I had just noticed this thread on Facebook today: https://www.facebook.com/groups/497105067092502/permalink/1886864508116544